src/react/components/StyleForm.jsx

```diff
@@ -0,0 +1,348 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Select, ColorPicker, SvgRadioButton, Preview } from 'oskari-ui';
+import { Form, Card, Space, Input, Row, Radio, InputNumber } from 'antd';
+import styled from 'styled-components';
+
+import { LineTab } from './styleform/LineTab';
+import { AreaTab } from './styleform/AreaTab';
+import { PointTab } from './styleform/PointTab';
+
+const formLayout = {
+    labelCol: { span: 24 },
+    wrapperCol: { span: 24 }
+}
+
+const lineIcons = {
```

Comment:
This file seems to do a bit too much but if you move these to the forms that actually need them it's going to reduce the amount of code here.

Reply:
Fixed.